app/src/main/java/com/example/moodup/viewmodel/HomeViewModel.kt

```diff
@@ -18,8 +18,8 @@ class HomeViewModel : ViewModel() {
     private val _comics = MutableLiveData<List<Comic>>()
     val comics: LiveData<List<Comic>> get() = _comics
 
-    private val publicKey = "1b1d366d594a5a142c7d2c04a5e0e17b"
-    private val privateKey = "d4aed0917e835baaf7ec60e084fc53f4bb011fff"
+    private val publicKey = "ddf99221f3ef736b4d3443c8cfd36c18"
```

Comment:
It's better to store API keys in local.properties
Please read this
https://medium.com/@vontonnie/secure-api-keys-in-android-projects-f8eb4839701d